app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills += ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills += ["Python", "SQL", "Django"]
+
+    def create_powerful_api(self) -> str:
+        print(f"{self.name} is creating an API...")
+
```

Comment:
Do you always add an empty line before `return`?
P.S: It's ok if you always do that, but you should stick to this rule (in this task you stick)

Reply:
Of course except situations, where the only 1 line in function is starting with `return` :)

Reply:
Yeah, I always add an empty line before return, maybe sometimes I forget do this. So is it better to do this, or without an empty line?

Reply:
Depending on the situation, I'd said. But if you like to do so - just do it everywhere. It's not a critical code-style issue :)